src/styles/blocks/page.scss

```diff
@@ -0,0 +1,4 @@
+.page {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
As mentioned in p3 of the main task: Display a stopwatch in the center of the page.
In your case, clock is not centered with a different scale of page
![Screenshot from 2022-11-14 10-50-50](https://user-images.githubusercontent.com/117041274/201616441-d684ba1a-7d13-44f7-abe3-efd4c70c0b8b.png)

I think it must be like this: 
```suggestion
.page {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
}
```
![Screenshot from 2022-11-14 10-52-51](https://user-images.githubusercontent.com/117041274/201616680-9f9a0c91-d4cc-4af9-8ada-3a76dbc4808d.png)

Reply:
Fix.